Xamarin.Essentials/Background/Background.android.service.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Android.App;
+using Android.Content;
+using Android.OS;
+
+namespace Xamarin.Essentials.Background
+{
+    [Service]
+    public class BackgroundService : Service
+    {
+        static bool isRunning;
+
+        public override IBinder OnBind(Intent intent)
+        {
+            return null;
```

Comment:
null? is this correct?

Reply:
These codes are copied from https://github.com/nblockchain/XamarinFormsBackground assessing accuracy of this code probably needs more testing